openid4vc-rest-api.json

```diff
@@ -11,15 +11,11 @@
     "clientAuthenticatorType": "client-secret",
     "secret": "${CLIENT_SECRET}",
     "redirectUris": [
-      "http://localhost:8080/*",
-      "https://kc-issuer.solutions.adorsys.com/*",
-      "http://kc-issuer.solutions.adorsys.com/*",
-      "http://localhost:8080",
+      "${ISSUER_BACKEND_URL}",
       "http://back.localhost.com/*"
     ],
     "webOrigins": [
-      "http://kc-issuer.solutions.adorsys.com",
-      "https://kc-issuer.solutions.adorsys.com"
+      "${ISSUER_BACKEND_URL}"
     ],
```

Comment:
Are you dependent on setting anything here? You're overwriting everything, right?